### DISCUSSION TITLE
Append spreadsheet values... The caller does not have permission

### DISCUSSION THREAD

**Discussion by ats3788** [mobizt/ESP-Google-Sheet-Client#28] [Q&A]
Reading works perfect but when I try to write to the Spreadsheet I get
Append spreadsheet values...
----------------------------
The caller does not have permission
I don't have a clue wehre to change this in the Google docs spreadsheet

**Comment by mobizt** [comment]
You can't edit the spreadsheet that wasn't created by the service account (this library).

The spreadsheet that only create using service account (this library) can be modified.

https://github.com/mobizt/ESP-Google-Sheet-Client/blob/1398e595744bcd0668da51913beccbff8bca3898/examples/Spreadsheets/Create/Create.ino#L181
